eng/python.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+  <Target Name="FindPythonWindows"
+          Condition="$([MSBuild]::IsOSPlatform(Windows)) and '$(PYTHON)' == ''"
+          Returns="$(PYTHON)">
+    <PropertyGroup>
+      <_PythonLocationScript>-c "import sys; sys.stdout.write(sys.executable)"</_PythonLocationScript>
+    </PropertyGroup>
+    <Exec Command="py -3 $(_PythonLocationScript) || py -2 $(_PythonLocationScript) || python3 $(_PythonLocationScript) || python2 $(_PythonLocationScript) || python $(_PythonLocationScript)"
```

Comment:
Our build requires python 3 so why are we using `py -2` and `python2`? That's basically allowing the build to proceed when we know it's going to fail. Better to error early and push developers to supported toolsets.

Reply:
I initially just pulled this from the old CoreCLR build script. I'll update it to only grab python3 installations.